newrelic/hooks/framework_azurefunctions.py

```diff
@@ -35,18 +33,12 @@ def intrinsics_populator(application, context):
     trigger_type = "http"
 
     website_owner_name = os.environ.get("WEBSITE_OWNER_NAME", None)
-    if not website_owner_name:
-        subscription_id = "Unknown"
-    else:
-        subscription_id = re.search(r"(?:(?!\+).)*", website_owner_name) and re.search(
-            r"(?:(?!\+).)*", website_owner_name
-        ).group(0)
-    if website_owner_name and website_owner_name.endswith("-Linux"):
-        resource_group_name = AZURE_RESOURCE_GROUP_NAME_RE.search(website_owner_name).group(1)
-    elif website_owner_name:
-        resource_group_name = AZURE_RESOURCE_GROUP_NAME_PARTIAL_RE.search(website_owner_name).group(1)
-    else:
-        resource_group_name = os.environ.get("WEBSITE_RESOURCE_GROUP", "Unknown")
+
+    subscription_id = "Unknown"
+    if website_owner_name and "+" in website_owner_name:
+        subscription_id = website_owner_name.split("+")[0] or "Unknown"
+
+    resource_group_name = os.environ.get("WEBSITE_RESOURCE_GROUP", "Unknown")
```

Comment:
The biggest issue here is that the `WEBSITE_RESOURCE_GROUP` environment variable is not guaranteed to be present.

Reply:
More specifically, it is not present in the Consumer plan and it is not always present in the Flex Consumer plan at the time we would need the value of this.